src/components/learning-guide-chart.coffee

```diff
@@ -97,25 +100,25 @@ module.exports = class LearningGuideChart
       .text("Show All #{guide.title}")
       .on('click', =>
         @displayTopic()
+        detailPane = document.querySelector('.learning-guide-chart .footer')
```

Comment:
Changing the React DOM outside of react worries me a bit. This is usually the cause of InvariantViolation Errors when transitioning to another page. Any reason code that removes the active class isn't in `@displayTopic` since that was created in the React component?